packages/sdk/src/message.ts

```diff
@@ -75,9 +75,12 @@ const blocsSchema = z.object({
   items: z.array(blocSchema),
 })
 
+/**
+ * @deprecated use `text` instead
```

Comment:
**note:** this is change 2; it will display the markdown message type as deprecated in most IDE